addons/website/static/tests/tours/drop_404_ir_attachment_url.js

```diff
@@ -1,53 +1,74 @@
 /** @odoo-module **/
 
+import { waitFor } from "@odoo/hoot-dom";
 import {
     changeOption,
-    dragNDrop,
     registerWebsitePreviewTour,
-} from '@website/js/tours/tour_utils';
+} from "@website/js/tours/tour_utils";
 
-registerWebsitePreviewTour('drop_404_ir_attachment_url', {
-    test: true,
-    url: '/',
-    edition: true,
-}, () => [
-    ...dragNDrop({
-        id: 's_404_snippet',
-        name: '404 Snippet',
-        groupName: "Images",
-    }),
+registerWebsitePreviewTour(
+    "drop_404_ir_attachment_url",
```

Comment:
I don't understand this whole tour change actually, is it still testing the same thing? Why is it dragging another unrelated snippet at the start, etc? Maybe needs to be into its own commit?